modules/face/include/opencv2/face/face_alignment.hpp

```diff
@@ -0,0 +1,58 @@
+/*This file is part of OpenCV project.
+It is subject to the license terms in the LICENSE file found in the top-level directory
+of this distribution and at http://opencv.org/license.html.
+*/
+
+#ifndef __OPENCV_FACE_ALIGNMENT_HPP__
+#define __OPENCV_FACE_ALIGNMENT_HPP__
+
+#include "opencv2/face/facemark.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/objdetect.hpp"
+#include "opencv2/imgproc.hpp"
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <vector>
+#include <fstream>
+#include <queue>
+#include <algorithm>
+#include <fstream>
+
+using namespace std;
+namespace cv{
+namespace face{
+    class FacemarkKazemi : public Facemark
```

Comment:
Add `CV_EXPORTS` here and in other cases.

BTW, Most part of includes above is implementation specific and should be moved into files under "src" subfolder.

Reply:
Thank you @alalek